documentation/authentication.md

```diff
@@ -0,0 +1,32 @@
+# Authentication
+
+The following mermaid flowchart depicts the authentication flow for the frontend.
+
+```mermaid
+---
+title: Authentication flow
+---
+flowchart TD
+    A[User logs in] --> B{User has previously accepted current terms and conditions?}
+
+    B -- No --> C{User accepts current terms and conditions?}
+    B -- Yes --> D{User exists in Cristin Person Register?}
+
+    C -- No --> E[User is logged out]
+    C -- Yes --> D
+
+    D -- No --> F{User accepts terms for being added to Cristin Person Register?}
```

Comment:
JE ville gjerne hatt C og F i samme steg for brukeren. Må ikke nødvendigvis gjenspeiles i denne modellen, men hadde nok vært en fordel.

Reply:
Har slått sammen disse til ett steg nå